### ISSUE TITLE
Feature request: Telegraf plugin: Make customized input/output configurations persistent across plugin updates

### ISSUE THREAD

**Issue by Grandma-Betty** [opnsense/plugins#3455]
**Important notices**
Before you add a new report, we ask you kindly to acknowledge the following:

- [x] I have read the contributing guide lines at https://github.com/opnsense/plugins/blob/master/CONTRIBUTING.md
- [x] I have searched the existing issues, open and closed, and I'm convinced that mine is new.
- [x] When the request is meant for an existing plugin, I've added its name to the title.

**Describe the solution you'd like**
I just discovered the OPNsense Telegraf plugin, which works great for monitoring the OPNsense instance itself. It is also possible to set custom input/ouput configurations via CLI in `/usr/local/etc/telegraf.conf` but unfortunately it is not possible to make those custom configurations persistent across Telegraf plugin updates, AFAIK. Which is a big limitation for such a great plugin which really makes sense to have it installed on the OPNsense instance itself for a centralized Telegraf interface within the network.

So in summary this is a feature request to make it possible to make customized Telegraf configurations persistent across Telegraf plugin updates.

**Additional context**
According to [this](https://forum.opnsense.org/index.php?topic=21431.0) post a potential feature request should probably already exist, but I was not able to find one. In this linked post it is also mentioned to make custom input/output configurations configurable via GUI, which would indeed be a nice feature too ;-)

**Comment by mimugmail** [comment]
/usr/local/etc/telegraf.conf.d

Just put your config there

**Comment by mimugmail** [comment]
https://github.com/opnsense/plugins/pull/2413/files
